segment_data/traffic_data.py

```diff
@@ -2,25 +2,41 @@
 import requests
 import datetime
 import pytz
+import json
 
 
 STRAVA_ACCESS_TOKEN = os.environ['STRAVA_ACCESS_TOKEN']
 TRAFFIC_URL = "https://www.strava.com/api/v3/segments/%s/all_efforts"
 CALIFORNIA_TIMEZONE = "US/Pacific"
+REDWOOD_SEGMENT_INPUT_FILE = 'redwood_segment_input.json'
 
 
-def get_traffic_count(segment_id, time_parameter):
+def build_segment_id_dict(segment_input_file):
```

Comment:
Isn't the file already in a usable format?  Looks like `trailways_id` is just the position in the array already, unless you anticipate it being more complicated in the future.  While it isn't the same as a dict exactly it seems like it would work without this next step so I'm curious why you'd do this extra work.